phidgets_imu/src/imu_ros_i.cpp

```diff
@@ -68,7 +68,8 @@ ImuRosI::ImuRosI(ros::NodeHandle nh, ros::NodeHandle nh_private)
 
     imu_publisher_ = nh_.advertise<ImuMsg>("imu/data_raw", 5);
     mag_publisher_ = nh_.advertise<MagMsg>("imu/mag", 5);
-    cal_publisher_ = nh_.advertise<std_msgs::Bool>("imu/is_calibrated", 5);
+    cal_publisher_ = nh_.advertise<std_msgs::Bool>("imu/is_calibrated", 5,
+                                                   true /* latched */);
```

Comment:
@mintar partial rollback regarding my comment "make it latched so one can check when it was calibrated the last time": the message has no timestamp. This means that one can only check if it was calibrated at some point in time.

Reply:
Anyway, it doesn't hurt to make it latched now. Maybe we can change the message definition at some point and add a timestamp to it.

Reply:
I think it's OK like it is now.